src/style/shared/number-input.less

```diff
@@ -30,6 +30,10 @@
     border: @hairline solid @highlight;
 }
 
+.number-input:disabled:hover {
+	border: none;
```

Comment:
This is a tab character. Thou shalt not use tab characters! Instead, you must configure your editor to _always_ create four-space tab stops.